ontology/proco.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: ontology_detail
+id: proco
+title: "Ontology of Process Chemistry"
+contact:
+  email: wes_schafer@merck.com
+  label: Wes Schafer
+  github: schaferw
+  orcid: 0000-0002-8786-1756
+description: OPC covers process chemistry, the chemical field concerned with scaling up laboratory syntheses to commercially viable processes.
+domain: biochemistry
+homepage: https://github.com/proco-ontology/PROCO
+products:
+  - id: proco.owl
+dependencies:
+  - id: ro
+  - id: obi
+  - id: chebi
+  - id: pato
+  - id: cheminf
+  - id: sbo
+tracker: https://github.com/proco-ontology/PROCO/issues
+license:
+  url: https://creativecommons.org/licenses/by/4.0/
+  label: CC BY 4.0
+activity_status: active
+repository: https://github.com/proco-ontology/PROCO
+preferredPrefix: PROCO
+---
+
+Previously named the Ontology of Process Chemistry (OPC), the PROcess Chemistry Ontology (PROCO) is aimed to be a community-based ontology in the domain of process chemistry, which is a branch of chemistry (including pharmaceutical chemistry) that studies the development and optimization of the production processes for chemical compounds, and the scaling up of laboratory chemical reactions into commercially viable routes. Key considerations are product quality, process robustness, economics, environmental sustainability, regulatory compliance and safety.
+
+PROCO was initiated as a collaborative project between Merck and Unversity of Michigan. Oliver He from Unversity of Michigan received a Merck fund and has been actively working with Wes Schafer at Merck on the PROCO development. Later our PROCO development team has been expanded to include more developers, including Anna Dunn and Zachary E. X. Dance.
```

Comment:
would be great to link these names with either ORCID identifiers or github handles - if there aren't any actionable ways to get in touch with these people then it's not super helpful to list them

Reply:
Common practice of attribution. I think you can figure out who is who based on the context. We can deal with such issues in more general tickets.